internal/cmd/beta/server/volume/attach/attach.go

```diff
@@ -0,0 +1,165 @@
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaasUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+const (
+	volumeIdArg = "VOLUME_ID"
+
+	serverIdFlag            = "server-id"
+	deleteOnTerminationFlag = "delete-on-termination"
+
+	defaultDeleteOnTermination = false
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ServerId            *string
+	VolumeId            string
+	DeleteOnTermination *bool
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "attach",
+		Short: "Attaches a volume to a server",
+		Long:  "Attaches a volume to a server.",
+		Args:  args.SingleArg(volumeIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Attaches a volume with id "xxx" to a server with id "yyy"`,
+				`$ stackit beta server volume attach xxx --server-id yyy`,
+			),
+			examples.NewExample(
+				`Attaches a volume with id "xxx" to a server with id "yyy" and delete on termination is enabled`,
+				`$ stackit beta server volume attach xxx --server-id yyy --delete-on-termination`,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			volumeLabel, err := iaasUtils.GetVolumeName(ctx, apiClient, model.ProjectId, model.VolumeId)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get volume name: %v", err)
+				volumeLabel = model.VolumeId
+			}
+
+			serverLabel, err := iaasUtils.GetServerName(ctx, apiClient, model.ProjectId, *model.ServerId)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get server name: %v", err)
+				serverLabel = *model.ServerId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to attach volume %q to server %q?", volumeLabel, serverLabel)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("attach server volume: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, volumeLabel, serverLabel, resp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), serverIdFlag, "Server ID")
+	cmd.Flags().BoolP(deleteOnTerminationFlag, "b", defaultDeleteOnTermination, "Delete the volume during the termination of the server. Defaults to false.")
```

Comment:
Isn't the default value already shown after the flag description when running the `--help` on the command?

Reply:
Hm, currently not but in comparison to the --nic-security flag for the network-interface command it is shown. I will take a look at it :+1:

Reply:
I see the difference: If the default value is false it is not printed, if the default value is true it is printed (automatically).

Shall we keep the info in the description or skip it?

Reply:
I think in this case we can skip it